src/pykeen/evaluation/evaluator.py

```diff
@@ -350,7 +355,7 @@ def _param_size_search(
     @staticmethod
     def _check_slicing_availability(model: Model, batch_size: int) -> None:
         # Test if slicing is implemented for the required functions of this model
-        if model.triples_factory.create_inverse_triples:
+        if model.create_inverse_triples:
```

Comment:
This is now a property of the model? Since we do not really "create" them, we could adjust the name, e.g., to `use_inverse_triples` / `add_inverse_relations`

Reply:
the model saves this property from the triples factory. actually it would be better to reorganize so this slicing availability function looks at the triples factory

Reply:
Will change this.

Reply:
7abfe27e